doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1091 @@
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.0.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.0.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.0.0`_ for the list of issues valid for the latest release.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Application development
+=======================
+
+* Fixed an issue with |NCS| Toolchain delimiter handling on MacOS, which could in special situations result in the build system not being able to properly locate the correct program needed.
+
+Boards
+======
+
+* Added necessary reset functionality for using the nRF52840 SoC on the Thingy:91 as a Bluetooth controller.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Bluetooth mesh
+--------------
+
+* Added :c:struct:`bt_mesh_sensor_srv` context to relevant callbacks and APIs to help resolve the associated sensor model instance.
+  For details, see `Bluetooth mesh samples`_ and `Bluetooth libraries and services`_.
+
+See `Bluetooth mesh samples`_ for the list of changes for the Bluetooth mesh samples.
+
+Matter
+------
+
+* The CHIP Tool controller is now the recommended controller for Matter.
+  The documentation about :ref:`ug_matter_configuring_controller` has been updated accordingly.
+  For more information about the CHIP Tool controller, read the :doc:`matter:chip_tool_guide` page in the Matter documentation.
+* Added the :ref:`matter_window_covering_sample` sample, based on the Matter upstream sample.
+
+See `Matter samples`_ for the list of changes for the Matter samples.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``25e241ebcbf11b1f63dbe25546b1f10219866ad0``.
+
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* Added the Binding cluster and Groupcast communication to the Light Switch sample.
+* Updated the nRF Connect platform by adding :file:`Kconfig.defaults`, :file:`Kconfig.mcuboot.defaults` and :file:`Kconfig.multiprotocol_rpmsg.defaults` files that contain the default configuration for all |NCS| samples.
+* Added support for Thread Synchronized Sleepy End Devices.
+
+Thread
+------
+
+* Added support for the Link Metrics and CSL Thread v1.2 features for the nRF53 Series devices.
+* Removed support for the :ref:`thread_architectures_designs_cp_ncp` architecture and the related tools.
+* Memory requirements page shows requirements based on the configuration used for certification instead of minimal configuration which has been removed.
```

Comment:
```suggestion
* :ref:`_thread_ot_memory` page shows requirements based on the configuration used for certification instead of minimal configuration, which has been removed.
```

Reply:
done